### ISSUE TITLE
Consider making the data capture "next" button more accessible

### ISSUE THREAD

**Issue by toolness** [18F/calc#480]
In #472 we add a "next" button with a visiual hint telling the user what they're going to be doing next:

> <img width="979" alt="screen shot 2016-07-28 at 3 19 36 pm" src="https://cloud.githubusercontent.com/assets/697848/17228005/c9077c04-54d6-11e6-8f9f-1f0c015ed063.png">

@hbillings mentioned in https://github.com/18F/calc/pull/472#discussion-diff-72786760 that it might be worth associating the help text with the "next" button at the aria level so that screen reader users have a better context for what pushing the button will do.

**Comment by toolness** [comment]
While I think this *might* be worth doing, it's also a bit of a hassle to test out, and it's unclear whether it'd be of lots of value.  It could also even potentially be confusing... I think we should hold off on this until we do user testing, at least--if vision-impaired users are confused by a "next" button that lacks more context, we can revisit this.

Removing from this milestone for now!
